Rakefile

```diff
@@ -26,7 +26,7 @@ namespace :test do
       Rake::Task["ci:setup:testunit"].execute
     end
     puts "Running Rails tests"
-    Rake::Task["test:run"].execute
```

Comment:
I have changed test:run to test and added rake test:all in .travis.yml to include coveralls task and jasmine tests. Although, all tests stop running once an error or failure shows up but I guess it doesn't shows up the warnings. Is it ok @jywarren